components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/OIDCSessionStateManager.java

```diff
@@ -44,4 +44,14 @@
      */
     Cookie addOPBrowserStateCookie(HttpServletResponse response);
 
+    /**
+     * Adds the browser state cookie with tenant qualified path to the response.
+     *
+     * @param response
+     * @return Cookie
+     */
+    default Cookie addOPBrowserStateCookie(HttpServletResponse response, String tenantDomain) {
+        return null;
```

Comment:
In default, we can call the old implementation addOPBrowserStateCookie(request)

Reply:
Added in c0887be